### PR TITLE
recommand sourcing bashrc in contributor's guide

### PR DESCRIPTION
**Pull request by emmanuelrouxfr**

### CODE REVIEW

File: how-to-contribute.md [hunk 1]
```diff
@@ -58,10 +58,13 @@ and setup a working environment.
 # rbenv-installer takes care of also installing ruby-build if `rbenv install` is not already available
 curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-installer | bash
 
-# Add rbenv to your bashrc to make it visible to your system
+# Add rbenv to your bashrc (and source it) to make it visible to your system each time you restart your machine (and now)
 echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
 echo 'eval "$(rbenv init -)"' >> ~/.bashrc
 
+# source your bashrc to make it visible to your system now
```
Comment (nathanpainchaud) [inline how-to-contribute.md:65]:
```suggestion
# source your bashrc to make it visible to your current shell
```

**Review by nathanpainchaud — CHANGES_REQUESTED** [review]
Thanks for the suggestion! Just two simple recommendations to avoid any possible confusion on the purpose of each command.

File: how-to-contribute.md [hunk 2]
```diff
@@ -58,10 +58,13 @@ and setup a working environment.
 # rbenv-installer takes care of also installing ruby-build if `rbenv install` is not already available
 curl -fsSL https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-installer | bash
 
-# Add rbenv to your bashrc to make it visible to your system
+# Add rbenv to your bashrc (and source it) to make it visible to your system each time you restart your machine (and now)
```
[the remark this answers is not in the document]
Comment (emmanuelrouxfr) [reply how-to-contribute.md:61]: All good, it's clearer !

**Review by nathanpainchaud — APPROVED** [review]
All good now! Thanks for the quick turn-around!

**Pull request merged**